.github/workflows/master.yml

```diff
@@ -35,10 +35,13 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 1
-      - name: Run smoke tests
-        env:
-          MASTER_BUILD: 1 # Will cause tests to use the true package artifacts, not local source.
-        run: make test
+      - name: Test + Push Release
```

Comment:
Once we merge to master, we actually set PUSH_IMAGES to allow pushed our new image to dockerhub.  This will allow future builds to use it as a reference cache.  I don't PUSH on PRs since they tend to involve more rapid changes.  We only want to push "canonical" images that are actually a result of merged code.

This won't currently trigger a deploy anywhere, this is mostly here now for the purpose of CI build caching.